.gitignore

```diff
@@ -5,3 +5,4 @@
 __pycache__/
 build/
 dist/
+*.egg-info
```

Comment:
Isn't \*.egg-info a directory?  If it's a directory it should have a trailing slash as well.